### ISSUE TITLE
Hippocampal sclerosis

### ISSUE THREAD

**Issue by DLewisSmith** [obophenotype/human-phenotype-ontology#6746]
**Preferred term label:**
Hippocampal sclerosis

**Synonyms**
Mesial temporal sclerosis

**Definition (free text, please give PubMed ID)**
Atrophy with gliosis of the hippocampus with loss of neurons and the normal cortical architecture.
24762203
**Parent term (use hpo.jax.org/app)**
Two parents:

Hippocampal atrophy HP:0410170
Gliosis HP:0002171

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Very common in epilepsies

**Your nano-attribution (ORCID)**
0000-0002-1735-8178

**Comment by pnrobinson** [comment]
added
